source/content/sites.md

```diff
@@ -63,48 +72,63 @@ The [Code tool](/code) on Dev/Multidev environments includes a Connection Mode t
 Learn how to use the [Pantheon workflow](/pantheon-workflow) to manage your site's code.
 
 ### Metrics
+
 Within the **<span class="glyphicons glyphicons-cardio"></span> Live** environment, the **<span class="glyphicons glyphicons-charts"></span> Metrics** tab provides insight into your site's traffic. Learn more in our guide on [Metrics in the Site Dashboard](/metrics).
 
 ![Site Metrics](../images/dashboard/metrics-graphs.png "Screenshot showing the Metrics tab of a Live site.")
 
 ### Status
+
 Pantheon provides static site analysis as a service for your site to make best practice recommendations on site configurations and to help detect common problems. This mechanism does not perform requests on your site, and in doing so avoids the observer effect. It's non-intrusive, so no installation or configuration is required. Finally, it's completely automated for consistent reports and results. Learn more about our Launch Check for
 [Drupal sites](/drupal-launch-check) or [WordPress sites](/wordpress-launch-check).
 
 ### Database / Files
+
 #### Clone
+
 Use this tool to clone database and files from an environment.
 
 #### Import
+
 You can import an archive of site files or a MySQL database via URL or file upload.
 
 #### Export
+
 With the Export tool, you can export the database or files from an environment to use in local development or a migration process. These exports are available for 365 days from the time of creation.
 
 #### Wipe
+
 Use this tool if you need to completely wipe your database and files for a single environment. Wiping completely resets the database and files and you will lose all content for that specific environment. For example, if you wipe the Dev environment, Test and Live are not affected. You will then need to import the database and files from a backup, clone them from another environment, or re-install Drupal or WordPress for that environment.
 
 Learn more about the [Pantheon Workflow](/pantheon-workflow).
 
 ### Errors
+
 All fatal PHP errors are shown on this tab. If any are found, recommendations are made. Learn more about [PHP Errors and Exceptions](/php-errors).
+
 <Alert title="Note" type="info">
+
```

Comment:
@rachelwhitton you had some whitespace on this line, which prevented it from being the empty line the MDX renderer needs to reset between Markdown and MDX. Tricky tricky.

Reply:
Ah, good to know though. Thank you!